docs/gatsby/api-reference/ocular-config.md

```diff
@@ -41,6 +41,19 @@ These are bullets on the home page, typically the main selling points of your fr
 | `desc`    | `String` | The text of the bullet point |
 | `image`   | `String` | The icon for the bullet. Typically `images/<icon>.svg`. Images are resolved in the `static` folder. |
 
+## ADDITIONAL_LINKS
+
+You can add additional links in the header section. These links will be added after the default menu.
+Each link entry has the following fields:
+
+| Field     | Type     | Name  |
+| ---       | ---      | ---   |
+| `label`    | `String` | Required. The displayed text in the header link. |
+| `href`    | `String` | Optional. The hyperlink you want to redirect to. |
+| `to`   | `String` | Optional. The path to a gatsby page |
```

Comment:
Is it either `href` or `to`? Maybe clarify?

Reply:
will do